giftless/util.py

```diff
@@ -52,3 +53,18 @@ def to_iterable(val: Any) -> Iterable:
     if isinstance(val, Iterable) and not isinstance(val, (str, bytes)):
         return val
     return (val,)
+
+
+def add_query_params(url: str, params: Dict) -> str:
```

Comment:
:+1: on this

When typing, would be better to specify a more accurate type for `params`, probably `Dict[str, str]` or if `urlencode` accepts int / float / other values, something like `Dict[str, Any]` - in any case the keys are always `str`.

Reply:
Ok I've just pushed a commit implementing this.  Did a little bit of research, urlencode docs says it should be a "dict made up of of str or bytes objects", however they don't strictly enforce this...

The result is that the keys actually can be of type int/float etc and the function still works as expected. i.e. if you run `add_query_params('https://a.b', {1.456: 9})` this executes smoothly giving the result `https://a.b?1.456=9` Not sure when you would design your query string like this, but is that enough to warrant the more restrictive type hints?  What are your thoughts?

Reply:
I think we don't have to enforce if `urlencode` doesn't, so I'd keep it at `Dict[str, Any]` as you did